### PR TITLE
few SOS related post-ICHEP fixes

### PR DESCRIPTION
**Pull request by cbotta**

### CODE REVIEW

File: TTHAnalysis/cfg/run_susyMultilepton_cfg.py [hunk 1]
```diff
@@ -444,13 +445,22 @@
...
 #    json = os.environ['CMSSW_BASE']+'/src/CMGTools/TTHAnalysis/data/json/Cert_271036-275125_13TeV_PromptReco_Collisions16_JSON.txt' # 4.0/fb
 #    json = os.environ['CMSSW_BASE']+'/src/CMGTools/TTHAnalysis/data/json/Cert_271036-275783_13TeV_PromptReco_Collisions16_JSON.txt' # 6.3/fb
-    json = os.environ['CMSSW_BASE']+'/src/CMGTools/TTHAnalysis/data/json/Cert_271036-276811_13TeV_PromptReco_Collisions16_JSON_NoL1T.txt' # 12.9/fb
-    processing = "Run2016B-PromptReco-v2"; short = "Run2016B_PromptReco_v2"; run_ranges = [(273150,276811)]; useAAA=True; # -v2 starts from 273150
+#    json = os.environ['CMSSW_BASE']+'/src/CMGTools/TTHAnalysis/data/json/Cert_271036-276811_13TeV_PromptReco_Collisions16_JSON_NoL1T.txt' # 12.9/fb #276811 ICHEP LastRun
+#    json = '/afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions16/13TeV/Cert_271036-279931_13TeV_PromptReco_Collisions16_JSON_NoL1T.txt' #24.5/fb
+    json = '/afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions16/13TeV/Cert_271036-280385_13TeV_PromptReco_Collisions16_JSON_NoL1T_v2.txt' #27.22/fb
```
Comment (peruzzim) [inline TTHAnalysis/cfg/run_susyMultilepton_cfg.py]: maybe these updates should not have been included in this round (fixing), but they're local and correct so it's fine

File: TTHAnalysis/python/plotter/susy-sos/fr-MET.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-change: met_pt  : met_genPt
-change: met_phi : met_genPhi
+change: met_pt\b  : met_genPt
```
Comment (peruzzim) [inline TTHAnalysis/python/plotter/susy-sos/fr-MET.txt]: please do a test and check that the functionality stays the same

File: TTHAnalysis/python/plotter/functions.cc [hunk 3]
```diff
@@ -619,6 +335,107 @@ float puw2016_nInt_12p9fb(float nInt, int var = 0) {
 //float puwMu17(int nVert) { return _puw_Mu17[nVert] * (2305428/29339.)*0.002/2.26; }
 
 
+
+
+//ttH stuff
+
+float ttH_MVAto1D_6_2lss_Marco (float kinMVA_2lss_ttbar, float kinMVA_2lss_ttV){
```
Comment (gpetruc) [inline TTHAnalysis/python/plotter/functions.cc]: these also should go in a separate functionsTTH.cc file

Comment (peruzzim) [inline TTHAnalysis/python/plotter/functions.cc]: Fixed in 616a91a

**Pull request merged**